src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/CosmosDBSinkConfig.java

```diff
@@ -0,0 +1,164 @@
+package com.microsoft.azure.cosmosdb.kafka.connect.sink;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+import org.apache.kafka.common.config.ConfigDef.Width;
+
+import java.util.Map;
+
+import com.microsoft.azure.cosmosdb.kafka.connect.TopicContainerMap;
+
+/**
+ * Contains settings for the Kafka ComsosDB Sink Connector
+ */
+
+@SuppressWarnings ({"squid:S1854", "squid:S2160"})  // suppress unneeded int *groupOrder variables, equals method
+public class CosmosDBSinkConfig extends AbstractConfig {
+
+    static final String COSMOS_CONN_ENDPOINT_CONF = "connect.cosmosdb.connection.endpoint";
+    private static final String COSMOS_CONN_ENDPOINT_DOC = "Cosmos DB endpoint URL.";
```

Comment:
sometimes we use just _**static final**_, and other times we use _**private static final**_
is this on purpose?

Reply:
yup, since the conf variable is used to specify the key in a Hashmap from which the source/sink config objects can be created. This is done in the unit tests and the source connector.